app/api/sessions.py

```diff
@@ -373,24 +373,27 @@ def notify_for_session(session, mail_override: Dict[str, str] = None):
     # Email for owner
     if session.event.get_owner():
         owner = session.event.get_owner()
-        send_email_session_state_change(owner.email, session, mail_override)
+        send_email_session_state_change(owner.email, session, mail_override, bcc)
         send_notif_session_state_change(
             owner, session.title, session.state, session.site_link, session.id
         )
+    # if bcc:
+    #     send_email_session_state_change(bcc, session, mail_override)
```

Comment:
@iamareebjamal sir, can i use this as it is sending mail to bcc.

Reply:
Why? It should send using above code already